src/lang/en-US.json

```diff
@@ -851,6 +892,7 @@
   "title_explorer": "Data Explorer",
   "title_meta": "Meta",
   "title_records": "Records",
+  "title_api": "The Opendota API: Advanced Dota 2 stats for your app",
```

Comment:
suggest removing "for your app" since that's not the only usage possible

Reply:
This is only used for SEO, and "app" will probably help.